valohai_yaml/objs/config.py

```diff
@@ -3,6 +3,7 @@
 from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple
 
 from valohai_yaml.lint import LintResult
+from ..excs import InvalidType
```

Comment:
Should this be an absolute import instead?

Reply:
Yes – please see https://github.com/valohai/valohai-yaml/pull/71; let's get that merged and I'll rebase this.